### ISSUE TITLE
minikube dashboard success,but could not connect to http://192.168.99.100:30000/

### ISSUE THREAD

**Issue by PeterSun15629150864** [kubernetes/minikube#2829]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

Please provide the following details:

**Environment**:

**Minikube version** (use `minikube version`):
- **OS** (e.g. from /etc/os-release):
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`):
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`):
- **Install tools**:
- **Others**:
The above can be generated in one go with the following commands (can be copied and pasted directly into your terminal):
```
minikube version
echo "";
echo "OS:";
cat /etc/os-release
echo "";
echo "VM driver": 
grep DriverName ~/.minikube/machines/minikube/config.json
echo "";
echo "ISO version";
grep -i ISO ~/.minikube/machines/minikube/config.json
```

**What happened**:

**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):

**Output of `minikube logs` (if applicable)**:

**Anything else do we need to know**:

**Comment by PeterSun15629150864** [comment]
<img width="760" alt="wx20180522-160800 2x" src="https://user-images.githubusercontent.com/27609099/40349971-d990ed3a-5dda-11e8-810f-041380b90999.png">
this is my kubernates pods,

**Comment by PeterSun15629150864** [comment]
<img width="1450" alt="wx20180522-161247 2x" src="https://user-images.githubusercontent.com/27609099/40350015-009fccf2-5ddb-11e8-90fe-752620693551.png">
when do minikube dashboard I can't open my page

**Comment by PeterSun15629150864** [comment]
my minikube version : minikube version: v0.25.0,
docker images i put in minikube : registry.docker-cn.com/googlecontainer/echoserver:1.4

**Comment by seborama** [comment]
Hi,

Can you provide the output of `minikube ip` and `kubectl get services --all-namespaces`?
And also `kubectl logs deployment.apps/kubernetes-dashboard -n kube-system`

Regards.

**Comment by PeterSun15629150864** [comment]
No, I just dont use minikube anymore,And I dont want you guys to send me email every single day


发自网易邮箱大师


On 09/27/2018 09:25，Gerard Braad<notifications@github.com> wrote：
The issue got resolved?

**Comment by gbraad** [comment]
I suspect this was related to an issue with your network setup, such as using a VPN.

**Comment by PeterSun15629150864** [comment]
yeep, I just dont wanna spend more time on this, And our team have started to use kubernetes 
 


发自网易邮箱大师


On 09/27/2018 09:29，Gerard Braad<notifications@github.com> wrote：

I suspect this was related to an issue with your network setup, such as using a VPN.

**Comment by shengbaodong** [comment]
this can be fix by exec:

kubectl create clusterrolebinding kube-system-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default

**Comment by goyalnikhil02** [comment]
Intially i was facing the issue while starting the minikube.
Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
So i follow the solution mention in https://github.com/kubernetes/minikube/issues/3882.It says simple .
Solved after doing sudo minikube delete, rm -rf ~/.kube ~/.minikube, then started over.

But now new issue coming. minikube is starting properly everytime i run "minikube start" command

But it is not accessible from by browser.
minikube ip have also changed ,so i try with new ip but still it is not accessible

**Comment by suranjanabasu** [comment]
I got the exact same issue. The solution was to uninstall both minikube and reinstall it. Used the following to uninstall
minikube stop; minikube delete
docker stop $(docker ps -aq)
rm -r ~/.kube ~/.minikube
sudo rm /usr/local/bin/localkube /usr/local/bin/minikube
systemctl stop '*kubelet*.mount'
sudo rm -rf /etc/kubernetes/
docker system prune -af --volumes

brew uninstall kubectl
rm /usr/local/bin/kubectl
brew link --overwrite kubernetes-cli

brew unlink kubernetes-cli && brew link kubernetes-cli

And the next time I installed, everything worked
